app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
what all this does: https://www.sitepoint.com/devise-authentication-in-depth/https://www.sitepoint.com/devise-authentication-in-depth/

database_authenticatable - users authenticate with their username/password and that's stored in the database
registerable - users can register, update, and de-register 
recoverable - allows them to recover passwords
rememberable - "remember me" cookies
trackable - tracks sign in count, IP addresses, and timestamps
validatable - validate email username password.